app/build.gradle

```diff
@@ -93,6 +93,7 @@ dependencies {
     implementation project(':sectionedrecyclerviewadapter')
     compileOnly 'org.projectlombok:lombok:1.18.4'
     annotationProcessor 'org.projectlombok:lombok:1.18.4'
+    implementation 'androidx.core:core-ktx:1.8.0'
```

Comment:
Luckily this will anyway be included because of the switch to Kotlin (I'll try to push my maintenance work soon).